File: exercicios em Python/q63.py
import os

print("\nEste programa determina o quadrante de pertencimente no plano cartesiano de um numero x, e y. ")
x = int(input("\n\nDigite o valor de x: "))
y = int(input("\nDigite o valor de y: "))
if (x > 0) & (y > 0):
        print("\n\nOs valores digitados x e y (",x,", ",y,") sao do primeiro quadrante.")

elif (x < 0) & (y > 0):
        print("\n\nOs valores digitados x e y (",x,", ",y,") sao do segundo quadrante.")

elif (x < 0) & (y < 0):
        print("\n\nOs valores digitados x e y (",x,", ",y,") sao do terceiro quadrante.")

elif (x > 0) & (y < 0):
        print("\n\nOs valores digitados x e y (",x,", ",y,") sao do quarto quadrante.")

else:
        print("\n\nOs dois numero estao em ponto de origem!")

print("\n\nObrigado por utilizar nosso sistema!")
os.system("pause")

File: exercicios em C/q63.cpp
#include <stdio.h>

int main(void){
	int x, y;
	printf("\n\nEste programa determina o quadrante de pertencimente no plano cartesiano de um numero x, e y. ");
	printf("\n\nDigite o valor de x: ");
	scanf("%d", &x);
	printf("\n\nDigite o valor de y: ");
	scanf("%d", &y);
	if (x > 0 & y > 0){
		printf("\n\nOs valores digitados x e y (%d, %d) sao do primeiro quadrante.", x, y);
	}
	else if (x < 0 & y > 0){
		printf("\n\nOs valores digitados x e y (%d, %d) sao do segundo quadrante.", x, y);
	}
	else if (x < 0 & y < 0){
		printf("\n\nOs valores digitados x e y (%d, %d) sao do terceiro quadrante.", x, y);
	}
	else if(x > 0 & y < 0){
		printf("\n\nOs valores digitados x e y (%d, %d) sao do quarto quadrante.", x, y);
	}
	else if(x == 0 & y == 0){
		printf("\n\nOs dois numero estao em ponto de origem!");
	}
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}
